lib/assertions.js

```diff
@@ -31,7 +31,13 @@ module.exports = expect => {
   );
 
   expect.addAssertion('<any> [not] to be <any>', (expect, subject, value) => {
-    expect(objectIs(subject, value), '[not] to be truthy');
+    expect.withError(
+      () => expect(objectIs(subject, value), '[not] to be truthy'),
+      err => {
+        err.label = 'should be';
```

Comment:
I completely forgot what the deal is with these labels, intuitively it should be able to work out really basic stuff like the label for `to be` without this extra faff.

Reply:
I'm a bit worried about this change as it adds something to a very hot path.

Reply:
We could consider re-writing this to:

```js
if (objectIs(subject, value) !== this.flags.not) {
  expect.fail({
    label: 'should be',
  })
}
```

Reply:
I know it takes a bit of beauty away, but maybe that is a fair trade-off.

Reply:
Ah, agreed. Fixed in 66b8452ce5034bc8e7597f4e85821ef18ecfecaf